### ISSUE TITLE
"building:roof" would be an interesting information for a future version ?

### ISSUE THREAD

**Issue by j3r3m1** [orbisgis/geoclimate#860]
There is the possibility to get the constructions consistent with the key value ["building:roof"](https://wiki.openstreetmap.org/wiki/FR:Tag:building=roof?uselang=fr). I have found a (couple of them)[https://www.openstreetmap.org/way/318131982] in Toulouse. It might be useful since it can be seen as a sun protection in the future (even though most of the time they probably do not have this use yet) and seen as potential solar panel potential.

@wurtzj @ebocher what do you think ?

**Comment by j3r3m1** [comment]
OK thank you for your answer. We do not consider them yet (they are not considered as buildings in GeoClimate).
However, it might be interesting to keep them at neighborhood scale or city scales for thermal comfort or energy applications using tools such as UMEP

**Comment by j3r3m1** [comment]
@biglimp ?

**Comment by biglimp** [comment]
Yes, depending on what kind of info that is included in the key values.
